include/diaspora.php

```diff
@@ -736,13 +736,30 @@ public static function url_from_contact_guid($fcontact_guid) {
 	 * @return The contact id
 	 */
 	private static function contact_by_handle($uid, $handle) {
+
+		// First do a direct search on the contact table
 		$r = q("SELECT * FROM `contact` WHERE `uid` = %d AND `addr` = '%s' LIMIT 1",
 			intval($uid),
 			dbesc($handle)
 		);
 
-		if ($r)
+		if ($r) {
+			//logger("Found contact ".$r[0]['id']." for user ".$uid." and handle ".$handle." - first try", LOGGER_DEBUG);
```

Comment:
Thanks, I already pointed it out in the previous displayed comment (check out the line numbers).